docs/GettingStarted.md

```diff
@@ -11,21 +11,23 @@ Table of Contents
 
 ## Installation
 
-If you are using the web installer, check out this [installation page](WebInstall.md).
+Follow [these instructions](UbuntuInstall.md) to install via debian packages.
 
-If you are installing from source, check out the [README](../README.md#installation).
+Follow [these instructions](BuildDigits.md) to build from source.
 
 ## Starting the server
 
 If you are using the web installer use the `runme.sh` script:
-
-    % cd $HOME/digits-2.0
-    % ./runme.sh
+```sh
```

Comment:
should this section be deleted and replaced with something that says that under the .deb package method the server is automatically started on local port x under user www-data, with digits log going into /xxx/yyy/digits.log